### DISCUSSION TITLE
Loop queries over an variable

### DISCUSSION THREAD

**Discussion by jkroepke** [grafana/grafana-infinity-datasource#869] [Ideas]
In grafana, we can do already loop by repeating Panls and Rows.

But in may case, i have to repeat a query, because I want to aggregate over multiple requests and generate sum at the end.

**Comment by yesoreyeram** [comment]
Refer "Iterating URLs in a loop" section in https://github.com/grafana/grafana-infinity-datasource/discussions/601. But the limitation is max 5 values can be iterated.

**Reply by jkroepke** [reply]
What the reason for the client-side limitation? I'm fine with DOS my own APIs
